interop-testing/src/test/java/io/grpc/testing/integration/MoreInProcessTest.java

```diff
@@ -249,7 +249,12 @@ public void onCompleted() {
         .onNext(StreamingInputCallRequest.getDefaultInstance());
 
     assertTrue(finishLatch.await(900, TimeUnit.MILLISECONDS));
-    assertEquals(Status.UNKNOWN, Status.fromThrowable(throwableRef.get()));
+    Status actualStatus = Status.fromThrowable(throwableRef.get());
+    Status expectedStatus = Status.UNKNOWN.withDescription(
+        "Internal Application Error @ task ServerCallListener(app).messagesAvailable");
+    assertEquals(expectedStatus.getCode(), actualStatus.getCode());
+    assertEquals(expectedStatus.getDescription(), actualStatus.getDescription());
```

Comment:
To allow some flexibility look for parts of the message that are important (startsWith and contains).

Reply:
On another hand, seeing the complete message is a good example of test-as-a-documentation.

I'll keep it as is for now (e.g. the whole message is important) - so if the message is extended, it'll be to the next dev to decide if whether the extension should be checked partially on in full.